src/logic/__test__/operator.test.js

```diff
@@ -0,0 +1,25 @@
+import operate from '../operator';
+
+describe('Unit test of the Calculator Logic', () => {
```

Comment:
Well done implementing tests for most of the operate.js functionalities 👏🏾 

- Please include tests for the modulus (%) operator as well...